### PR TITLE
Update defender-advanced-threat-protection.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Updated with new URL for Defender portal in several locations. 
Updated instructions for on- and offboarding according to the new portal.  
Other smaller markdown corrections, like removing extra spaces between list items.

### CODE REVIEW

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -110,37 +116,37 @@ The installer package will check if the following components have already been i
...
 > [!IMPORTANT]
 > - The Microsoft Defender for Endpoint configuration file contains sensitive information which should be kept secure.
 > - If your target collection contains down-level devices that require MMA, and you use the instructions for onboarding using MDE Client, then the down-level devices won't be onboarded. The optional **Workspace key** and **Workspace ID** fields are used for onboarding down-level devices, but if they aren't included then the policy will fail on down-level clients.
 
-
 #### Onboard the up-level devices
 
 1. In the Configuration Manager console, navigate to **Administration** > **Client Settings**.
 1. Create custom Client Device Settings or go to the properties of the required client setting and select **Endpoint Protection**
 1. For **Microsoft Defender for Endpoint Client on Windows Server 2012 R2 and Windows Server 2016** setting, The default value is set as **Microsoft Monitoring Agent (legacy)** which needs to be changed to **MDE Client (recommended)**.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:139]:
```suggestion
1. Create custom Client Device Settings or go to the properties of the required client setting and select **Endpoint Protection**.
1. For **Microsoft Defender for Endpoint Client on Windows Server 2012 R2 and Windows Server 2016** setting, the default value is set as **Microsoft Monitoring Agent (legacy)** which needs to be changed to **MDE Client (recommended)**.
```

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md [hunk 2]
```diff
@@ -213,29 +219,29 @@ When you onboard devices to Microsoft Defender for Endpoint with Configuration M
...
-
 ### <a name="bkmk_any_os"></a> Onboard devices with any supported operating system to Microsoft Defender for Endpoint (recommended)
+
  You can onboard devices running any of the [supported operating systems](#bkmk_os) to Microsoft Defender for Endpoint by providing the configuration file, **Workspace key**, and **Workspace ID** to Configuration Manager.
 
 #### Get the configuration file, workspace ID, and workspace key
 
-1. Go to the [Microsoft Defender for Endpoint online service](https://security.microsoft.com/) and sign in.
-1. Select **Settings**, then select **Onboarding** under the **Endpoints** heading.
+
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:228]:
```suggestion
```

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md [hunk 3]
```diff
@@ -266,72 +271,66 @@ If your target collection contains both up-level and down-level devices, or if y
...
 1. Choose **Microsoft Endpoint Configuration Manager current branch and later** for the deployment method.
-1. Select **Download package**.
+1. Select **Download onboarding package**.
+
+   :::image type="content" source="media/5229962-onboarding-configuration.png" alt-text="Screenshot of onboarding configuration file download." lightbox="media/5229962-onboarding-configuration.png":::
+
 1. Download the compressed archive (.zip) file and extract the contents.
     > [!NOTE]
     > The steps have you download the onboarding file for Windows 10 and 11 but this file is also used for up-level Server operating systems.
+1. Change the operating system to either **Windows 7 SP1 and 8.1** or **Windows Server 2008 R2 Sp1** from the list.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:285]:
```suggestion

1. Change the operating system to either **Windows 7 SP1 and 8.1** or **Windows Server 2008 R2 Sp1** from the list.
```

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md [hunk 4]
```diff
@@ -266,72 +271,66 @@ If your target collection contains both up-level and down-level devices, or if y
...
-    - **Microsoft Defender ATP Agent Health**: Percentage of computer clients reporting status for their Microsoft Defender for Endpoint agent  
-
-        - **Healthy** - Working properly  
-
-        - **Inactive** - No data sent to service during time period  
-
-        - **Agent state** - The system service for the agent in Windows isn't running  
-
-        - **Not onboarded** - Policy was applied but the agent hasn't reported policy onboard  
+1. In the Configuration Manager console, navigate **Monitoring** > **Security** and then select **Microsoft Defender ATP**.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:306]:
```suggestion
1. In the Configuration Manager console, navigate to **Monitoring** > **Security** and then select **Microsoft Defender ATP**.
```

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md [hunk 5]
```diff
@@ -266,72 +271,66 @@ If your target collection contains both up-level and down-level devices, or if y
...
+1. Review the Microsoft Defender for Endpoint dashboard.
+    - **Microsoft Defender ATP Agent Onboarding Status**: The number and percentage of eligible managed client computers with active Microsoft Defender for Endpoint policy onboarded
+    - **Microsoft Defender ATP Agent Health**: Percentage of computer clients reporting status for their Microsoft Defender for Endpoint agent
+        - **Healthy** - Working properly
+        - **Inactive** - No data sent to service during time period
+        - **Agent state** - The system service for the agent in Windows isn't running
+        - **Not onboarded** - Policy was applied but the agent hasn't reported policy onboard
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:313]:
```suggestion
    - **Microsoft Defender ATP Agent Onboarding Status**: The number and percentage of eligible managed client computers with active Microsoft Defender for Endpoint policy onboarded.
    - **Microsoft Defender ATP Agent Health**: Percentage of computer clients reporting status for their Microsoft Defender for Endpoint agent:
        - **Healthy** - Working properly.
        - **Inactive** - No data sent to service during time period.
        - **Agent state** - The system service for the agent in Windows isn't running.
        - **Not onboarded** - Policy was applied but the agent hasn't reported policy onboard.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md (hunk 2 above)
Comment (baardhermansen) [reply memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:228]: Hi @JohanFreelancer9 
Wouldn't your suggestion break with rule regarding headings?
[Rule MD022](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md022)

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md (hunk 3 above)
Comment (baardhermansen) [reply memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:285]: Sorry, but i don't see how this is beneficial? The step belongs to the ones above, not as a separate list?

**Comment by BalaDelli** [comment]
#sign-off

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md (hunk 2 above)
Comment (JohanFreelancer9) [reply memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:228]: @baardhermansen, only one blank line is needed around the headings. If we don't remove the blank line as suggested, there will be two blank lines (lines 227 and 228, whereas only one is required.

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md (hunk 3 above)
Comment (JohanFreelancer9) [reply memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:285]: Hi @baardhermansen, yes you are correct. I apologize, my mistake. I'll remove the suggestion. Many thanks.

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md (hunk 2 above)
Comment (baardhermansen) [reply memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:228]: @JohanFreelancer9  Aha, i misread the diff...

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by baardhermansen** [comment]
Ping @BalaDelli for a review.

**Review by BalaDelli — APPROVED** [review]

**Comment by BalaDelli** [comment]
#sign-off

**Pull request closed without merging**